spec/system/user_interacts_with_pdf_viewer_spec.rb

```diff
@@ -223,5 +225,23 @@
       click_button "Global suggestions"
       expect(page).to have_content("Some random string longer than 15 chrs")
     end
+
+    context "when the user exports own suggestions", js: true do
+      around do |example|
+        original_setting = ActionController::Base.allow_forgery_protection
+        ActionController::Base.allow_forgery_protection = true
+        example.run
+        ActionController::Base.allow_forgery_protection = original_setting
+      end
```

Comment:
Why do we need this?
is not the user clicking on the export button? shoudn't the authenticity_token be present already?